Makefile

```diff
@@ -286,15 +274,19 @@ uninstall-crds: manifests kustomize ## Uninstall CRDs from the K8s cluster speci
 installation-scripts:
 	perl -pi -e "s/version=.*/version=$(VERSION)/g" install/scripts/install-kubectl-plugins.sh
 
-.PHONY: installation-manifests
-installation-manifests: manifests kustomize
+.PHONY: installation-manifests-gcp
+installation-manifests-gcp: manifests kustomize
+	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
```

Comment:
This is duplicated in each one of these makefile targets. Also by removing the original target we have broken the `prepare-release` target. Prefer to keep one target here to avoid Makefile sprawl.